boa/src/builtins/date/tests.rs

```diff
@@ -1006,7 +1008,8 @@ fn date_proto_set_utc_full_year() -> Result<(), Box<dyn std::error::Error>> {
 
     let actual = forward_dt_utc(
         &mut engine,
-        "dt = new Date(Date.UTC(2020, 06, 08, 09, 16, 15, 779)); dt.setUTCFullYear(2012, 8, 10); dt",
+        "dt = new Date(Date.UTC(2020, 06, 08, 09, 16, 15, 779)); dt.setUTCFullYear(2012, 8, 10); \
+         dt",
```

Comment:
This to me, seems very weird and remind me of the C++ macros (that i hate). for me is less readable.